src/app/[locale]/topics/[topicId]/(topic)/components/flashcard/FlashcardContent.tsx

```
@@ -7,133 +7,98 @@ import {
-                        <LayoutGrid className="mr-2 h-4 w-4" />
-                        <span>Browse</span>
-                        {mode === 'browse' && <Check className="ml-auto h-4 w-4" />}
-                    </DropdownMenuItem>
-
-                    <DropdownMenuItem onSelect={() => handleModeSelect('learning')}>
-                        <GraduationCap className="mr-2 h-4 w-4" />
-                        <span>Learning</span>
-                        {mode === 'learning' && <Check className="ml-auto h-4 w-4" />}
-                    </DropdownMenuItem>
+                    {selectableItems.map((item) => (
+                        <DropdownMenuItem onSelect={() => handleModeSelect(item)}>
```

Biome (2.1.2)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)